pkg/backend/endpoint_types.go

```diff
@@ -12,7 +12,7 @@ type PodEndpoint struct {
 	// Pod's IP.
 	IP string
 	// Pod's container port.
-	Port int64
+	Port int32
```

Comment:
both int64 and int32 is good enough, since port range is at most 0-65535.
We can choose whatever that minimizes the conversation between k8s api/aws sdk

Reply:
The aws-sdk-v2 is now using int32 type of ports hence I have defined our endpoint port to match it.